src/components/UserMasterPanel.js

```diff
@@ -5,7 +5,6 @@ import { withModulesManager, useTranslations, TextInput, PublishedComponent, com
 import { CLAIM_ADMIN_USER_TYPE, ENROLMENT_OFFICER_USER_TYPE } from "../constants";
 // import { fetchRegionDistricts } from "../actions";
```

Comment:
Why we have here commented line? (It is not a part of this PR but it could be good to verify this)